src/gui/optionsdlg.h

```diff
@@ -60,6 +60,8 @@ namespace Ui
 class OptionsDialog: public QDialog
 {
     Q_OBJECT
+    using ThisType = OptionsDialog;
```

Comment:
I added this because `ThisType` is shorter and cleaner than `OptionsDialog` and since I'm used to make such aliases, I've put in into class declaration as I always do. So, should I move it into the constructor body?